docker/depends/Dockerfile

```diff
@@ -1,20 +1,12 @@
-ARG R_VERSION="4.0.2"
+ARG R_VERSION="4.1"
+ARG FROM_IMAGE="rocker/tidyverse"
```

Comment:
Being able to change the source image is the point of this PR. Whether that’s a good idea is a key question, but I don’t think it can be removed without changing approaches — this is what allows the same script to either build the image fresh from the upstream Rocker image or be updated from an existing pecan/develop.

My hope is that in the cases where we want to use this, the gain in build speed will offset the cost of the (thin) extra layers, but please do burst my bubble if that won’t be true.

Reply:
ARG is not written as a layer in the docker image so we don't have to worry about. This use of the ARGS allows us to customize the depends image to point to any version any base image. These are then used in all subsequent builds. If you do a `docker history pecan/depends:develop` you will not see any of the ARGS.

Removing the stretch section below saves 4KB and one layer.